pygfx/resources/_texture.py

```diff
@@ -42,7 +42,9 @@ class Texture(Resource):
         automatically determined from the data.
     colorspace : str
         If this data is used as color, it is interpreted to be in this
-        colorspace. Can be "srgb" or "physical". Default "srgb".
+        colorspace. Can be "srgb", "physical", "yuv420p", or "yuv444p". Default "srgb".
+    colorrange : str
+        For YUV textures, this is either "limited", or "full". See... TODO BY MARK
```

Comment:
I'm half-wondering if this should just be a suffix to the colorspace, e.g. "yuv420p-full"

Reply:
I just feel like we would be making the whole `yuvj420p` vs `yuv420p` mistake that took ffmpeg like 4 years to undo recently.

They now explicitely have the pixel format and the color range as two separate parameters.